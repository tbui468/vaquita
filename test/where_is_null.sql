create database sol;
open sol;

create table planets (id int key, name string, moons int);

insert into planets (id, name) values (1, "Mars");
insert into planets (id, name, moons) values (2, "Earth", 1);
select * from planets where moons is null;

close sol;
drop database sol;
exit;
