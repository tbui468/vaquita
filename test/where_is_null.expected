created database sol
opened database sol
created table planets
inserted 1 record(s) into planets
inserted 1 record(s) into planets
id, name, moons
1, Mars, null
closed database sol
dropped database sol
